compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -794,17 +794,28 @@ object Trees {
   }
 
 
+  abstract class GenericImport[-T >: Untyped](implicit @constructorOnly src: SourceFile)
```

Comment:
I think I would simply call it ImportOrExport since an Export isn't really an Import.